spec/models/vm/retirement_management_spec.rb

```diff
@@ -163,16 +161,12 @@
   end
 
   it "#raise_retirement_event with current user" do
-    user = FactoryGirl.create(:user_with_group, :userid => 'freddy')
     event_name = 'foo'
-    event_hash = {:vm => @vm, :host => @vm.host, :type => "ManageIQ::Providers::Vmware::InfraManager::Vm",
-                  :retirement_initiator => "user", :userid => 'freddy'}
-    options = {:zone => @vm.my_zone}
+    event_hash = {:userid => user, :vm => @vm, :host => @vm.host, :type => "ManageIQ::Providers::Vmware::InfraManager::Vm"}
+    options = {:zone => @vm.my_zone, :user_id => user.id}
```

Comment:
:see_no_evil: Wow, this is inconsistent

Reply:
I was looking at the `user.id` wondering if it was supposed to be the id or userid.

Reply:
Well it was originally the ```userid``` and then it got changed to ```user_id``` so I'mma die on this hill cause I think that it's right, albeit unfortunate, the way it is here.